devops  alona

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request
+import subprocess
+
+app = Flask(__name__)
+
+@app.route('/system-info', methods=['GET'])
+def system_info():
+    command = request.args.get('command')
+    if not command:
+        return "No command ", 400
+
+    try:
+        result = subprocess.run(command, shell=True, capture_output=True, text=True, check=True)
+        return result.stdout
+    except subprocess.CalledProcessError as e:
```

Comment:
I believe except will terminate the process with no chance to recover, we need to pormp a message that the command wasn't successful and some additional data, without terminating the program